CHANGELOG.en-US.md

```diff
@@ -13,1372 +13,165 @@ timeline: true
-- Fix wrong behavior of Upload when uid is 0. [#7269](https://github.com/ant-design/ant-design/issues/7269)
-
-## 2.12.6
-
-`2017-08-11`
-
-- Fix Collapse arrow position in IE9/10/11. [af9dc73](https://github.com/ant-design/ant-design/commit/af9dc73eef2aac3e68fdfc99f9231153437f5068)
-- Fix empty Table scrollBar's position. [#7142](https://github.com/ant-design/ant-design/issues/7142)
-- Fix RangePicker Input's style of height. [47c5de2](https://github.com/ant-design/ant-design/commit/47c5de29f370a77fc19a6a8eda8ac57476f312a6)
-- Card
-  - Improve Card.Grid style of collapse border. [807b573](https://github.com/ant-design/ant-design/commit/807b573bff43bc71115afeb086f565561807a32d)
-  - Fix the `no-animation` type's className. [#7185](https://github.com/ant-design/ant-design/issues/7185)
-- Fix Input Safari text align. [#7135](https://github.com/ant-design/ant-design/issues/7135)
-- Layout
-  - Fix stretched layout by content. [500b222](https://github.com/ant-design/ant-design/commit/500b2225567f03397d9faec5f4e60a8f35fc4d28)
-  - Fix the collapse trigger's position. [f689ede](https://github.com/ant-design/ant-design/commit/f689ede0fa836dd0d99f4e4d96e0c43d0ff19742)
-- Fix Upload doesn't handle some error scenarios which response is not a string. [#6818](https://github.com/ant-design/ant-design/issues/6818)
+## 3.0.0
+
+`2017-12-04`
+
+### Major Changes
+
+- New [color system](https://ant.design/docs/spec/colors#Color-Palettes), We changed our primary color from 『`#108EE9`』 to 『`#1890FF`』, we called the new primary color "拂晓蓝" which means the color of the sky at daybreak.
+- New design of components.
+- Increased the base font size from 12px to 14px.
+- Changed default locale to en_US.
+- Support React 16.
+- Better TypeScript support.
+- New [List](https://ant.design/components/list/) component.
+- New [Divider](https://ant.design/components/divider/) component.
+- 30 New [icons](https://ant.design/components/icon/).
+
+### Breaking Changes
+
+- Card's `noHovering` has renamed to `hoverable`，and it's default value now is `true`.
+- Added new Grid breakpoints. [#7230](https://github.com/ant-design/ant-design/pull/7230)
+- Form `getFieldDecorator`'s `exclusive` option was removeed.
+- Added `Form.createFormField`, and you must use it to wrap field data returned in `option.mapPropsToFields`:
+
+```diff
+import { Form } from 'antd';
+
+Form.create({
+  mapPropsToFields() {
+    return {
+-     name: { value: 'antd' },
++     name: Form.createFormField({ value: 'antd' }),
+    };
+  },
+})
+```
+
+- Improved global reset style, if you encounter style problem after upgrading, you can try import our v2 compatible style.
+
+```javascript
+import 'antd/style/v2-compatible-reset';
+```
+
+Or use less
+
+```less
+@import '~antd/style/v2-compatible-reset.css';
+```
+
+- Since we changed default locale to en_US, LocalProvide is not required for English users now.
```

Comment:
LocalProvide is not required for English users now. -> LocalProvider is not required any more for English users.